components/gitpod-protocol/scripts/publish.js

```diff
@@ -36,6 +36,6 @@ fs.writeFileSync(path.join(pckDir, "package.json"), JSON.stringify(pck, undefine
 const tag = qualifier.substr(0, qualifier.lastIndexOf("."));
 
 child_process.execSync(
-    ["yarn", "--cwd", pckDir, "publish", "--tag", tag, "--access", "public", "--ignore-scripts"].join(" "),
+    ["yarn", "--cwd", pckDir, "publish", "--tag", tag, "--access", "public", "--ignore-scripts", "--network-timeout", "100000"].join(" "),
```

Comment:
This one has been failing recently on main with:

```
error Couldn't publish package: "https://registry.yarnpkg.com/@gitpod%2fgitpod-protocol: ESOCKETTIMEDOUT"
```

<img width="1688" alt="image" src="https://user-images.githubusercontent.com/5501705/216306955-bb2767a9-ffed-4cb8-8ddd-d542de1f425c.png">